common/src/test/java/com/gentics/mesh/test/junit/Assert.java

```diff
@@ -21,4 +23,17 @@ public static void assertException(GenericRestException exception, HttpResponseS
 		assertArrayEquals("The i18n parameters did not match the expected values", i18nParams, exception.getI18nParameters());
 	}
 
+	/**
+	 * Assert that the numeric values of two numbers are equal.
+	 * @param expected the expected value
+	 * @param actual the actual value
+	 */
+	public static void assertNumberValueEquals(Number expected, Number actual) {
```

Comment:
This class should not be extended. I plan to get rid of it in favor of AssertThat assertions.